lib/v8.js

```diff
@@ -15,5 +15,20 @@
 'use strict';
 
 var v8binding = process.binding('v8');
-exports.getHeapStatistics = v8binding.getHeapStatistics;
+var smalloc = require('smalloc');
+
+var data = smalloc.alloc(5, smalloc.Types.Uint32);
```

Comment:
Yeah. It depends on the GC to cleanup the memory, and since the object sticks around indefinitely then it'll show up. Though we could amend the second commit to free the memory on exit.